src/service/fileMaker.ts

```diff
@@ -4,20 +4,69 @@ import * as fs from '../util/fs';
 import { log } from '../util/logger';
 import { interpolate } from '../util/string';
 import { getTimestampString } from '../util/ts';
+import MakeOptions from '../domain/MakeOptions';
 import Configuration from '../domain/Configuration';
 import FileExtensions from '../enum/FileExtensions';
 import { getMigrationPath } from '../migration/service/knexMigrator';
 
 const MIGRATION_TEMPLATE_PATH = path.resolve(__dirname, '../../assets/templates/migration');
 const CREATE_TABLE_CONVENTION = /create_(\w+)_table/;
+const SOURCE_TYPE_STUBS = {
+  sql: ['create_up.stub', 'create_down.stub'],
+  javascript: ['create_js.stub', 'update_js.stub'],
+  typescript: ['create_ts.stub', 'update_ts.stub']
+};
+
+/**
+ * Get template stub directory path.
+ *
+ * @param {string} basePath
+ * @param {string} stubPath
+ * @returns {string | null}
+ */
+export async function publish(config: Configuration): Promise<string[]> {
+  const templateFiles = [];
+  const templateBasePath = path.join(config.basePath, '/stubs');
+  const templateBasePathExists = await fs.exists(templateBasePath);
+  const templates = SOURCE_TYPE_STUBS[config.migration.sourceType] || [];
+
+  if (!templateBasePathExists) {
+    log(`Template base path does not exist, creating ${templateBasePath}`);
+
+    await fs.mkdir(templateBasePath, { recursive: true });
+  }
+
+  log(`Templates to be moved: ${templates}`);
+
+  for (const template of templates) {
+    if (await fs.exists(path.join(templateBasePath, template))) {
+      log(`File already exists: ${template}`);
```

Comment:
How do we let the users know about this?

Reply:
@cham11ng Can we also write a test where the stub file already exists and we try to publish?

Reply:
I'll update the console log for this. Sure, will add it.

Reply:
@mesaugat Check the description. I've added the screenshot.